<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reveiver</class>
 <widget class="QMainWindow" name="Reveiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="4">
      <widget class="QPushButton" name="Btn_readatttach">
       <property name="text">
        <string>查看附件</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Btn_back">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label3">
       <property name="text">
        <string>收件人：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QLabel" name="label_Title">
       <property name="text">
        <string>Receive：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QLabel" name="label_sender">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="Btn_delete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_attachname">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="5">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>附件：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Btn_reply">
       <property name="text">
        <string>回复</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label8">
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="Btn_readatttach_2">
       <property name="text">
        <string>下载附件</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>发件人：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="5">
      <widget class="QLabel" name="label_reciever">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="6">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
